### PR TITLE
Package tezos-rust-libs.1.1

### PR DESCRIPTION
**Pull request by dannywillems**
### `tezos-rust-libs.1.1`

This is a new version of tezos-rust-libs which is going to be used by [bls12-381](https://gitlab.com/dannywillems/ocaml-bls12-381) for a JavaScript support using jsoo. The new version of bls12-381 (tagged 0.4.1) is coming right after.
The library is already pinned [here](https://gitlab.com/dannywillems/ocaml-bls12-381/-/blob/master/bls12-381-unix.opam#L30).
It doesn't change that much compared to 1.0, except it bumps up a dependency bls12-381 requires: rustc-bls12-381.

Released with @pirbo who can approve this PR.

**Something important**: the required rust version is 1.44.0 (`cargo vendor` has been used with rust 1.44.0). However, I am not sure the CI checks this. It is not possible yet to change to a specific rust version in OPAM.
I am not against adding a version check in the build (plus an additional error message). However, it might not pass the CI here. 

Tezos: all rust dependencies and their dependencies
---
* Homepage: https://www.tezos.com/
* Source repo: git+https://gitlab.com/tezos/tezos-rust-libs.git
* Bug tracker: https://gitlab.com/tezos/tezos-rust-libs/issues

---
:camel: Pull-request generated by opam-publish v2.0.2

### CODE REVIEW

File: packages/tezos-rust-libs/tezos-rust-libs.1.1/opam [hunk 1]
```diff
@@ -0,0 +1,18 @@
+opam-version: "2.0"
+synopsis: "Tezos: all rust dependencies and their dependencies"
+maintainer: "contact@tezos.com"
+authors: "Tezos devteam"
+license: "multiple"
+homepage: "https://www.tezos.com/"
+bug-reports: "https://gitlab.com/tezos/tezos-rust-libs/issues"
+depends: ["conf-rust"]
+build: ["cargo" "build" "--target-dir" "target" "--release"]
```
Comment (kit-ty-kate) [inline packages/tezos-rust-libs/tezos-rust-libs.1.1/opam:9]: The previous version has this instead:
```
  ["mkdir" ".cargo"]
  ["mv" "cargo-config" ".cargo/config"]
  ["cargo" "build" "--target-dir" "target" "--release"]
```
Is it not needed anymore?

Comment (pirbo) [reply packages/tezos-rust-libs/tezos-rust-libs.1.1/opam:9]: Argh, my mistake and well caught, Yes this is still needed...

Comment (dannywillems) [reply packages/tezos-rust-libs/tezos-rust-libs.1.1/opam]: Nice catch. I fixed in a second commit, with the same message than https://github.com/ocaml/opam-repository/pull/17880

Comment (dannywillems) [reply packages/tezos-rust-libs/tezos-rust-libs.1.1/opam]: Mmh, wait. Not sure it is required. The file is at the correct place: see https://gitlab.com/tezos/tezos-rust-libs/-/commit/f0a269087fe86dbe1cf471bb76e42fb4a481e226.

**Review by pirbo — APPROVED** [review]

**Comment by mseri** [comment]
Thanks

**Pull request merged**